README.md

```diff
@@ -1 +1,89 @@
-# ovirt-ansible-engine-setup
+oVirt Engine Setup
+==================
+
+Install required packages for oVirt Engine deployment, generates answerfile
+and runs engine-setup with it.
+Optionally the role can update oVirt engine packages.
+
+Target Systems
+--------------
+
+* engine
+
+Requirements
+------------
+
+ * Environment with configured repositories.
+ * Ansible version 2.3
+
+Role Variables
+--------------
+
+By default engine-setup uses answer file specific for version of oVirt,
+based on ``ovirt_engine_setup_version`` parameter. You can specify own answer file
+as ``ovirt_engine_setup_answer_file_path``.
+
+| Name                            | Default value         |  Description                                              |
+|---------------------------------|-----------------------|-----------------------------------------------------------|
+| ovirt_engine_setup_version            | UNDEF                 | Allowed versions: [3.6, 4.0, 4.1, 4.2] |
+| ovirt_engine_setup_product_type       | ovirt-engine          | Type of product 'ovirt-engine' or 'rhvm'  | 
+| ovirt_engine_setup_package_list       | [ovirt-engine-dwh-setup,ovirt-engine-extension-aaa-ldap] |  List of extra packages to be installed on engine apart from ovirt-engine package. |
+| ovirt_engine_setup_answer_file_path   | UNDEF                 | Path to custom answerfile for `engine-setup`. |
+| ovirt_engine_setup_db_host            | localhost             | IP address or host name of a PostgreSQL server for Engine database. By default the database will be configured on the same host as Engine. |
+| ovirt_engine_setup_db_port            | 5432                  | Engine database port. |
+| ovirt_engine_setup_db_name            | engine                | Engine database name. |
+| ovirt_engine_setup_db_user            | engine                | Engine database user. |
+| ovirt_engine_setup_db_password        | UNDEF                 | Engine database password. |
+| ovirt_engine_setup_dwh_db_host        | localhost             | IP address or host name of a PostgreSQL server for DWH database. By default the DWH database will be configured on the same host as Engine. |
```

Comment:
Why not add a special section `Data Warehouse database` similar to below ISO section?